src/types.d.ts

```diff
@@ -26,6 +26,7 @@ type Question = {
 	upvotes: number;
 	user: number;
 	viewer_vote: null | boolean;
+	fetchedAnsPages?: number[] 
```

Comment:
Make this fetchedAPages to keep consistency and add a comment to it identifying its purpose